src/test/java/io/strimzi/kafka/bridge/http/ConsumerGeneratedNameTest.java

```diff
@@ -54,24 +110,26 @@ void createConsumerNameNotSet(VertxTestContext context) throws InterruptedExcept
                 .as(BodyCodec.jsonObject())
                 .sendJsonObject(json, ar -> {
                     context.verify(() -> {
+                        LOGGER.info("Verifying that consumer name is created with '-' plus random hashcode");
                         assertThat(ar.succeeded(), is(true));
                         HttpResponse<JsonObject> response = ar.result();
+                        LOGGER.info("Response code from the Bridge is " + response.statusCode());
                         assertThat(response.statusCode(), is(HttpResponseStatus.OK.code()));
                         JsonObject bridgeResponse = response.body();
-                        String consumerInstanceId = bridgeResponse.getString("instance_id");
-                        name = consumerInstanceId;
-                        assertThat(consumerInstanceId.startsWith("kafka-bridge-consumer-"), is(true));
+                        consumerInstanceId = bridgeResponse.getString("instance_id");
+                        LOGGER.info("Consumer instance of the consumer is " + consumerInstanceId);
+                        assertThat(consumerInstanceId.startsWith("-"), is(true));
```

Comment:
why changing this condition? Name is not set, bridge id is not set ... the instance id starts with "kafka-bridge-consumer-"

Reply:
changed the line 55 and let the bridge.id == null and not the empty string ("")